edb/edgeql/compiler/expr.py

```diff
@@ -479,15 +479,24 @@ def compile_GlobalExpr(
         qry = qlast.SelectQuery(result=qlast.Path(steps=[obj_ref]))
         return dispatch.compile(qry, ctx=ctx)
 
+    default = glob.get_default(ctx.env.schema)
+
+    # If we are compiling with globals suppressed but still allowed, always
+    # treat it as being empty.
```

Comment:
I'm not sure about this.  Semantically, migrations are just EdgeQL scripts, so technically I should be allowed to `SET GLOBAL` in them.

Reply:
Fair enough, but can you please add a test then in case we somehow allow them to be parsed but forget to actually support things properly.

Reply:
Sorry, merged without reading, will do a follow up